Python/perf_jit_trampoline.c

```diff
@@ -174,8 +174,8 @@ static const uint8_t DwarfDataRel = 0x30;
 typedef struct {
     unsigned char version;
     unsigned char eh_frame_ptr_enc;
-    unsigned char	fde_count_enc;
-    unsigned char	table_enc;
+    unsigned char       fde_count_enc;
+    unsigned char       table_enc;
```

Comment:
Looks like these were tabs replaced with multiple spaces. Can you just insert one space instead?

```suggestion
    unsigned char fde_count_enc;
    unsigned char table_enc;
```